### PR TITLE
updated .pre-commit-config.yaml

### PR DESCRIPTION
**Pull request by sairevanth0007**
## Description

I have added the [blacken-docs](https://github.com/adamchainz/blacken-docs) hook to [.pre-commit-config.yaml file](https://github.com/strawberry-graphql/strawberry/blob/main/.pre-commit-config.yaml) as it was posted as an good first issue.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [x] Enhancement/optimization
- [ ] Documentation

## Issues Fixed or Closed by This PR

*

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the CONTRIBUTING document.
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -36,3 +36,8 @@ repos:
     hooks:
       - id: black
         exclude: ^tests/codegen/snapshots/python/
+
+  - repo: https://github.com/asottile/blacken-docs
```
Comment (patrick91) [inline .pre-commit-config.yaml:40]:
```suggestion
  - repo: https://github.com/adamchainz/blacken-docs
```

just noticed the docs have the wrong url, I'll update it for you!

**Review by patrick91 — APPROVED** [review]
Thank you so much!

File: docs/_test.md [hunk 2]
```diff
@@ -36,7 +36,7 @@ class X:
 This is probably not implemented in the best way, but for now it works:
 
 ```python
-import ^[info](strawberry)
+import [info](strawberry)
```
Comment (patrick91) [inline docs/_test.md:39]: @sairevanth0007 was this failing? 

the `^` was on purpose :D

**Comment by sairevanth0007** [comment]
@patrick91 I am doing the correct changes? Do I need to change anything?

File: docs/_test.md (hunk 2 above)
Comment (sairevanth0007) [reply docs/_test.md]: yes i had got this in failing errors.

Comment (patrick91) [reply docs/_test.md]: uhm, gotcha, I'll take a better look later tonight or tomorrow

**Review by patrick91 — APPROVED** [review]
I've made some small updates to get CI green, but other than that it looks great! thanks!

**Comment by botberry** [comment]
Thanks for contributing to Strawberry! 🎉 You've been invited to join
the Strawberry GraphQL organisation 😊

You can also request a free sticker by filling this form: https://forms.gle/dmnfQUPoY5gZbVT67

And don't forget to join our discord server: https://strawberry.rocks/discord 🔥

**Comment by sairevanth0007** [comment]
> I've made some small updates to get CI green, but other than that it looks great! thanks!

Thank you.

**Pull request merged**